package.json

```diff
@@ -36,5 +40,32 @@
     "eslint": "~2.2.0",
     "jbj": "^4.1.1",
     "mocha": "^2.4.5"
-  }
+  },
+  "gitHead": "95ecddff31e6a55f3ecc956811e9c6cd6ff9fe2d",
```

Comment:
The below part seems to be added by npm itself, but I don't understand why, and it should not be useful.